modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/metrics/PlanningCacheMetricsTest.java

```diff
@@ -56,20 +56,20 @@ public void plannerCacheStatisticsTest() throws Exception {
         MetricSet metricSet = metricManager.enable(SqlPlanCacheMetricSource.NAME);
 
         try {
```

Comment:
Now we have 1 additional read be prepare with no dynamic params or with all parameters set.